.github/workflows/lint-markdown.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: Check markdown syntax
+
+"on":
+  push:
+    paths:
+      - '**.md'
+      - .github/workflows/lint-markdown.yml
+
+jobs:
+  lint-markdown-syntax:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: reviewdog/action-markdownlint@v0
```

Comment:
This is some github builtin markdown linter?
I have no idea what it does ...
But yes, in general, getting some consistency in what markdown we put in there is definitely something I appreciate.

Reply:
@garloff these are repositories in general. So something like "reviewdog/action-markdownlint" translates to github.com/reviewdog/action-markdownlint